config/control-plane/kustomization.yaml

```diff
@@ -90,6 +87,26 @@ patches:
 # Note1: Now as the 'patchesJson6902' is deprecated, the direct use of the 'PatchTransformer' is the only way to change a resource namespace to something different from the value configured by the global namespace transformer.
 # Note2: the 'grafana_dashboard' label could be applied using a standard JSON patch, but a namespace must be patched here and it's better to have all related customizations in a single place. 
 transformers:
```

Comment:
I think the comments above need to be updated. We don't use `the global namespace transformer` anymore.

We also had a comment from SRE wrt. usage of the `kyma-system` namespace [here](https://github.tools.sap/kyma/management-plane-charts/pull/858) (see heading `Dashboard CMs`). Do we need to alter that namespace anyway (maybe also in a follow-up)?

Reply:
But was this agreed upon? But let's do this in a followup not here because I would prefer not to change any namespaces in this PR to make sure everything is working fine as is first